tests/Command/Bus/HandlerLocatedCommandBusTest.php

```diff
@@ -13,17 +13,18 @@
 use GpsLab\Component\Command\Bus\HandlerLocatedCommandBus;
 use GpsLab\Component\Command\Command;
 use GpsLab\Component\Command\Handler\Locator\CommandHandlerLocator;
+use GpsLab\Component\Command\Exception\HandlerNotFoundException;
 use PHPUnit\Framework\TestCase;
 
 class HandlerLocatedCommandBusTest extends TestCase
 {
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject|CommandHandlerLocator
+     * @var \PHPUnit\Framework\MockObject\MockObject|CommandHandlerLocator
      */
     private $locator;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject|Command
+     * @var \PHPUnit\Framework\MockObject\MockObject|Command
```

Comment:
Since we now use namespaces, it is better to declare them in the `use` section.
```php
use PHPUnit\Framework\MockObject\MockObject;

class HandlerLocatedCommandBusTest extends TestCase
{
    /**
     * @var MockObject|Command
     */
    private $command;

    // ...
}
```